lib/mapper.js

```diff
@@ -108,6 +113,53 @@ function formatCompany(company){
   return ret;
 }
 
+ /**
+ * Formats location data for use with intercom
+ *
+ * https://doc.intercom.io/api/#user-model
```

Comment:
siiiiiiiiiiick thanks for linking to docs, it's super helpful to people who maintain this code down the line